lte/protos/session_manager.proto

```diff
@@ -789,55 +813,76 @@ enum SMSessionFSMState {
-  uint32 max_pkt_filter_supported = 20;    //Maximum no. of supported packet filters
-  bool pdu_session_req_always_on = 21;     //Always-on PDU session requested TS-24.501 9.11.4.4
-  string sm_pdu_dn_req = 22;               //SM PDU DN request container. Max size up to 255. TLV and 9.11.4.16.
-  M5GSMCause m5gsm_cause = 23;
-  QosRules requested_qos_rules = 24; //TS-24501 - 9.11.4.13 minimum 7 and max 65538, variable length.
+  // Define enum TS-29.571 - section 5.4.3
+  AccessType access_type = 5;
+  // pei - permanent equipment ID. Variable length of 16 bytes. TS-23.003 - 6.4 Like IMEI or IMEISV
+  string imei = 6;
+  // Lenght is 10 bytes + 2 bytes country code generic public subscription id.
+  string gpsi = 7; 
+  // up to 6 byte. TS-23.003 - 28.15.3
+  string user_location_info = 8;
+  // length 16 bytes as UUID "Universally Unique IDentifier" TS-29.571 to ietf rfc 4122
+  string pcf_id = 9;
+  // Variable length of 20 bytes. TS-23.003 - 28.8 & TS-23.501 5.9.8
+  repeated LadnServiceArea ladn_service_area = 10; 
+  // Subscription For PDU Session Status Notification true or false
+  bool pdu_session_status_notification_sub = 11;
+  // TS-29-502 - 6.1.6.3.10 
+  DnnSelectionMode dnn_selection_mode = 12;
+  // TS-29.571 5.6.
+  string trace_requirement = 13;
+  // Length is 3 bytes, TS-24.501 9.11.4.7
+  bytes integrity_protection_max_data_rate = 14;
+  PduSessionType pdu_session_type = 15;
+  SscMode ssc_mode = 16;
+  // max size go up to 15 bytes. TLV and 9.11.4.1.
+  M5GSMCapability m5g_sm_capability = 17;  // max size go up to 15 bytes. TLV and 9.11.4.1.
+  // Maximum no. of supported packet filters
+  uint32 max_pkt_filter_supported = 18;
+  // Always-on PDU session requested TS-24.501 9.11.4.4
+  bool pdu_session_req_always_on = 19;
+  // SM PDU DN request container. Max size up to 255. TLV and 9.11.4.16.
+  string sm_pdu_dn_req = 20; 
+  M5GSMCause m5gsm_cause = 21;
+  // TS-24501 - 9.11.4.13 minimum 7 and max 65538, variable length.
+  QosRules requested_qos_rules = 22;
+  TeidSet gnode_endpoint = 23;
+  TeidSet upf_endpoint = 24;
+  // Procedure transaction identity
+  bytes procedure_trans_identity = 25;
 }
 
 message SetSMSessionContext {
-  CommonSessionContext common_context = 1;  //Common message for 4g, 5g and WiFi.
+  // Common message for 4g, 5g and WiFi.
+  CommonSessionContext common_context = 1;
   RatSpecificContext   rat_specific_context = 2;
 }
 
-//TODO This supposed to be empty message and the element will be removed later
+// TODO This supposed to be empty message and the element will be removed later
 message SmContextVoid {
   string output = 1;
 }
 
-//SET API for 3 procedures request from AMF to SMF
+// SET API for 3 procedures request from AMF to SMF
         service AmfPduSessionSmContext {
                 rpc SetAmfSessionContext (SetSMSessionContext) returns (SmContextVoid);
         }
 
 ////////// SET MESSAGE PROTO DEFINATION OF SMF TO AMF ON RESPONSE /////////////
```

Comment:
typo DEFINATION. Also all these ////... look too much....

Reply:
Fixed it.